docs/rfcs/rfc-009-graphql-ui-poc.md

```diff
@@ -0,0 +1,112 @@
+# GraphQL Graph App PoC
+
+## Problem
+
+The [GraphQL Architect](https://grandstack.io/docs/graphql-architect-overview/#:~:text=GraphQL%20Architect%20is%20a%20graph,Neo4j%20Desktop%20Graph%20Apps%20Gallery) for usage in the Neo4j Desktop is due to be replaced. As an initial step a proof of concept of a successor shall be established.
+
+## Proposed Solution
+
+_Name suggestions:_  
+Explorer  
+Architect  
+Editor  
+UI  
+Frame  
+(Already taken: Studio, Playground, GraphiQL)
+
+### User audience
+
+The `@neo4j/graphql` lib's user audience are full stack developers who want a GraphQL API quickly without worrying about the database. This UI can further include developers wanting to experiment in general as well as experiment with/edit the introspected schema.
```

Comment:
Question: Could this tool also be beneficial for our own development of `@neo4j/graphql` ?

Reply:
Do you mean as an "alternative" (or replacement) to the `test-api.ts`? I had that in mind when writing the pitch but was hesitant to put it as a requirement as we were unsure about the exact user audience for the tool. We will discuss the user audience later today and I'll bring this up. Good thou that you mention it!

Reply:
We just had a chat with Helene and yes, we should definitely build a v1/PoC that is to our own benefit as well. It is likely that other dev's will like to use the app if we ourselves are (and want to) using it too.